src/_locales/en-US.json

```diff
@@ -62,6 +62,7 @@
   "playback-label-thumbs-up": "Thumbs Up",
   "playback-label-volume-down": "Decrease Volume",
   "playback-label-volume-up": "Increase Volume",
+  "playback-label-info-track": "Notify current track",
```

Comment:
The tests will fail without an empty translation string in all the other language files.